src/command/dev/router/config.rs

```diff
@@ -287,6 +287,7 @@ impl RouterConfigReader {
             tp.spawn(move || loop {
                 raw_rx
                     .recv()
+                    .expect("could not watch router configuration file")
```

Comment:
Is there a reason we need two expects here?

Reply:
Yes! This is because `.recv()` return a `Result<T, RecvError>`,  and `T` in this case is `Result<(), RoverStdError>`